packages/autorest.typescript/src/generators/static/packageFileGenerator.ts

```diff
@@ -185,6 +185,7 @@ function regularAutorestPackage(
     packageInfo.devDependencies["@azure/dev-tool"] = "^1.0.0";
     packageInfo.scripts["build"] =
       "npm run clean && tsc && dev-tool run bundle && npm run minify && mkdirp ./review && npm run extract-api";
+    packageInfo.scripts["extract-api"] = "dev-tool run extract-api";
```

Comment:
May I know if we need to switch all actions to dev-tool?

Reply:
For the JS monorepo we would prefer the consistent and centralized dev-tool command.  It adds support for the subpath exports.